### PR TITLE
Reword/restyle Glossary Definition for Speed Index

### PR DESCRIPTION
**Pull request by airbr**
Hello, I am studying Speed Index right now and I came across this Glossary Definition. I thought that it could be slightly improved, with a little context about why this metric exists, and if it is appropriate to link to the two major projects using the metric in their audits.

I also think visually the image is large for the entry, so I opted to put all the text above the image with a description to further help.

Hope this proposed change resonates. I believe the information is correct at least, style wise maybe an improvement.

### CODE REVIEW

File: files/en-us/glossary/speed_index/index.md [hunk 1]
```diff
@@ -8,11 +8,14 @@ tags:
   - Web Performance
 ---
 
-**Speed Index** (SI) is a page load performance metric that shows you how quickly the contents of a page are visibly populated. It is the average time at which visible parts of the page are displayed. Expressed in milliseconds and dependent on the size of the viewport, the lower the score, the better.
+**Speed Index** (SI) is a page load performance metric measuring how quickly the contents of a page are visibly populated. Speed Index is dependent on size of the viewport and expressed in milliseconds: the lower amount of time the better the score.
 
-![Calculation of SpeedIndex](speedindex.png)
+Speed Index was introduced to address issues with other milestones and metrics and provide a gauge of real user experience. Speed Index has been implemented in several common audits including [WebPageTest](https://github.com/WPO-Foundation/webpagetest-docs/blob/main/src/metrics/SpeedIndex.md) and [Lighthouse](https://github.com/paulirish/speedline).
```
Comment (wbamberg) [inline files/en-us/glossary/speed_index/index.md:13]:
```suggestion
Speed Index was introduced to address issues with other milestones and metrics and provide a gauge of real user experience. Speed Index has been implemented in several common audits including [WebPageTest](https://github.com/WPO-Foundation/webpagetest-docs/blob/main/src/metrics/SpeedIndex.md) and [Lighthouse](https://github.com/paulirish/speedline).
```

File: files/en-us/glossary/speed_index/index.md [hunk 2]
```diff
@@ -8,11 +8,14 @@ tags:
...
 
-![Calculation of SpeedIndex](speedindex.png)
+Speed Index was introduced to address issues with other milestones and metrics and provide a gauge of real user experience. Speed Index has been implemented in several common audits including [WebPageTest](https://github.com/WPO-Foundation/webpagetest-docs/blob/main/src/metrics/SpeedIndex.md) and [Lighthouse](https://github.com/paulirish/speedline). 
+
+Speed Index specifically is calculated by what percent of the page is visually complete at every 100ms interval until the page is visually complete. The overall score is a sum of the individual 10 times per second intervals of the percent of the screen that is not visually complete.
 
-The calculation calculates what percent of the page is visually complete at every 100ms interval until the page is visually complete. The overall score, the above the fold metric, is a sum of the individual 10 times per second intervals of the percent of the screen that is not visually complete.
+*Diagram showing how above the fold content can load before the page load event and is measured by Speed Index*:
```
Comment (wbamberg) [inline files/en-us/glossary/speed_index/index.md:17]:
```suggestion

_Diagram showing how above the fold content can load before the page load event and is measured by Speed Index_:
```

File: files/en-us/glossary/speed_index/index.md [hunk 3]
```diff
@@ -8,11 +8,15 @@ tags:
...
+**Speed Index** (SI) is a page load performance metric measuring how quickly the contents of a page are visibly populated. Speed Index is dependent on size of the viewport and expressed in milliseconds: the lower amount of time the better the score.
 
-![Calculation of SpeedIndex](speedindex.png)
+Speed Index was introduced to address issues with other milestones and metrics and provide a gauge of real user experience. Speed Index has been implemented in several common audits including [WebPageTest](https://github.com/WPO-Foundation/webpagetest-docs/blob/main/src/metrics/SpeedIndex.md) and [Lighthouse](https://github.com/paulirish/speedline).
+
+Speed Index specifically is calculated by what percent of the page is visually complete at every 100ms interval until the page is visually complete. The overall score is a sum of the individual 10 times per second intervals of the percent of the screen that is not visually complete.
```
Comment (teoli2003) [inline files/en-us/glossary/speed_index/index.md:15]:
```suggestion
Speed Index is calculated by what percent of the page is visually complete at every 100ms interval until the page is visually complete. The overall score is a sum of the individual 10 times per second intervals of the percent of the screen that is not visually complete.
```

**Comment by airbr** [comment]
Thank you both reviewers for reviewing this document.

I should add, that WebPageTest contains documentation about this topic that I did not link, I just linked the speed index github repo.

I hope that some content I proposed is able to be used. Appreciate MDN, its a great resource, thank you

File: files/en-us/glossary/speed_index/index.md [hunk 4]
```diff
@@ -4,11 +4,15 @@ slug: Glossary/Speed_index
...
 
-![Calculation of SpeedIndex](speedindex.png)
+Speed Index was introduced to address issues with other milestones and metrics and provide a gauge of real user experience. Speed Index has been implemented in several common audits including [WebPageTest](https://github.com/WPO-Foundation/webpagetest-docs/blob/main/src/metrics/SpeedIndex.md) and [Lighthouse](https://github.com/paulirish/speedline).
+
+Speed Index is calculated by what percent of the page is visually complete at every 100ms interval until the page is visually complete. The overall score is a sum of the individual 10 times per second intervals of the percent of the screen that is not visually complete.
 
-The calculation calculates what percent of the page is visually complete at every 100ms interval until the page is visually complete. The overall score, the above the fold metric, is a sum of the individual 10 times per second intervals of the percent of the screen that is not visually complete.
+
```
Comment (teoli2003) [inline files/en-us/glossary/speed_index/index.md:13]:
```suggestion
```

**Review by teoli2003 — APPROVED** [review]
👍

**Comment by teoli2003** [comment]
Thanks a lot!

This PR looks good to me.

Congratulations upon your first merged commit here! Welcome aboard! ✌️ 

If you're looking for something else to help out with, we have a [contributor board](https://github.com/orgs/mdn/projects/25), and another useful place to look is the list of issues labeled [`good first issue`](https://github.com/mdn/content/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22).

**Pull request merged**